### PR TITLE
Switching to using generations TFM

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
Checking this in to unblock CI.

File: src/Microsoft.Data.Sqlite/project.json [hunk 1]
```diff
@@ -1,62 +1,40 @@
...
+        "System.IO.FileSystem": "4.0.1-*",
+        "System.Linq": "4.0.1-*",
+        "System.Reflection": "4.1.0-*",
+        "System.Resources.ResourceManager": "4.0.1-*",
+        "System.Runtime": "4.0.21-*",
+        "System.Runtime.Extensions": "4.0.11-*",
+        "System.Runtime.Handles": "4.0.1-*",
+        "System.Runtime.InteropServices": "4.0.21-*",
+        "System.Text.Encoding": "4.0.11-*",
+        "System.Threading.Tasks": "4.0.11-*"
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite/project.json]: Will this break UWP? We were explicitly using the RTMed versions...

Comment (pranavkm) [inline src/Microsoft.Data.Sqlite/project.json]: We added `netcore50` as a TFM in a followup PR.

Comment (pranavkm) [inline src/Microsoft.Data.Sqlite/project.json]: https://github.com/aspnet/Microsoft.Data.Sqlite/commit/1aaf4edf71dc677909d43fadd1f5fd83b4d2e86b

**Pull request merged**